packages/path_provider/path_provider_android/example/README.md

```diff
@@ -0,0 +1,8 @@
+# path_provider_example
```

Comment:
nit: The typical pattern of the example apps of plugins would name this `path_provider_android_example`

Reply:
I don't think its that important to keep the pattern unless one of our tests uses this pattern to look for test

Reply:
There shouldn't be anything that depends on it. I'm going to leave this as-is since I'm not 100% sure that changing it to something that doesn't match the existing native code won't cause issues. (Normally the name of everything would be auto-generated to be right, but I wanted to leave exactly the same native runner that we already know works rather than re-generate, as this is intended to be as close to a file move as possible.)